README.md

```diff
@@ -144,11 +144,10 @@ You can deploy this project using:
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Odette Gleen**
+👤 **Lucas Bermudez**
 
-- GitHub: [@Gleennkar](https://github.com/Gleennkar)
-- Twitter: [@gleeny24](https://twitter.com/twitterhandle)
-- LinkedIn: [@gleeny](https://www.linkedin.com/in/gleeny-nkar-aa3917182)
+- GitHub: [@Oklukeok](https://github.com/Oklukeok)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/lucas-bermudez)
 
```

Comment:
- [ ] Great job adding devise. In your `ReadMe`, under the `Install` section, you say `No Install Needed, `but I think I had to bundle install, create database and migrate it before i could start the server locally. Please kindly all these steps to enable a user run or start your app with ease.